### DISCUSSION TITLE
self-hosted: How to install jsonnet-bundler properly?

### DISCUSSION THREAD

**Discussion by maxbrunet** [renovatebot/renovate#13569] [Request Help] [answered]
`binarySource=install` being "globalOnly", it does not seem I can do the following:

```json
{
  "jsonnet-bundler": {
    "binarySource": "install"
  }
}
```

And I do not want to set `binarySource=install` for any other manager.

I run Renovate in Kubernetes, so I do not concider `binarySource=docker` as an option due to:
1. the security implications of Docker-in-Docker
2. the Kubernetes Dockershim is deprecated, distributions like AWS EKS are migrating to use `containerd` instead

(Maybe support for a rootless container solution like [Podman](https://podman.io/) could be added? Edit: Failed experiment #4387 🤔)

So I currently run the job this way: `install-tool jb v0.4.0 && renovate`

Is there anything against adding this 8MB binary to [renovatebot/docker-renovate-full](https://github.com/renovatebot/docker-renovate-full) or could I open a PR?

**Comment by rarkins** [comment, marked as answer]
We can add it to the full image and a PR would be welcome

**Reply by maxbrunet** [reply]
Opened [renovatebot/docker-renovate-full#396](https://github.com/renovatebot/docker-renovate-full/pull/396)!

**Reply by subesokun** [reply]
@rarkins Not sure if this is related, but could it be that the Flux CLI needs to be installed into the full image too? Because we've a failing job when Renovate tries to update the Flux `gotk-components.yaml` file. The [flux datasource docu ](https://docs.renovatebot.com/modules/manager/flux/#additional-information)says something about installing Flux CLI on your own or enabling `binarySource=install`. But the [docu here](https://docs.renovatebot.com/self-hosted-configuration/#binarysource) doesn't list Flux yet as supported via binary source install and I'm also not sure if this is the correct way to go ...

**Reply by viceice** [reply]
It's supported via `binarySource=install` but the docs is missing this. @HonkingGoose Can you update those?

**Reply by HonkingGoose** [reply]
> It's supported via `binarySource=install` but the docs is missing this. @HonkingGoose Can you update those?

See PR:

- https://github.com/renovatebot/renovate/pull/13924 😉
